docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerCloud.java

```diff
@@ -63,7 +63,12 @@
     public final String serverUrl;
     private int connectTimeout;
     public final int readTimeout;
-    public final String version;
+    /**
+     * Confuses users and no real use cases.
+     * TODO remove
```

Comment:
1.0 or when i decide to break it :) Nobody depends on this plugin AFAIK and job-dsl is not implemented for plugin.